parts/k8s/kubernetesconfigs.sh

```diff
@@ -392,7 +392,7 @@ configAddons() {
 }
 
 configGPUDrivers() {
-    sh nvidia-drivers-$GPU_DV --silent --accept-license --no-drm --dkms --utility-prefix="${GPU_DEST}" --opengl-prefix="${GPU_DEST}"
+    sh /usr/local/nvidia/nvidia-drivers-$GPU_DV --silent --accept-license --no-drm --dkms --utility-prefix="${GPU_DEST}" --opengl-prefix="${GPU_DEST}" || exit $ERR_GPU_DRIVERS_START_FAIL
```

Comment:
Can we also add the full path to the curl output destination in `parts/k8s/kubernetesinstalls.sh`?